Week1/Section3/Exercise5/Source5.cpp

```diff
@@ -38,10 +38,10 @@ int main()
 
 	int Sum = 0;
 	std::cout << "numbers entered: [";
-	for (int x : MyArray)
+	for (int i : MyArray)
```

Comment:
I'm sorry I didn't explain myself better hehe, but in this case, the `i` is also not acceptable. `i` is for an index in a for-loop. In this case `i` is not actually an index, but the actual object in the array, so in a traditional for loop, this `i` would be the equivalent of `MyArray[i]` which should have a descriptive name.

Reply:
You are right it doesn't give us an index, it give us the element 👌
changed for a more descriptive name